Bottom: 3afb51c6c6d08a52ef72ca238aa3e1406caad752
Top:    fc28baf3514fe78424490d143b1112b8f900d448
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 12:49:22 +0800

Refresh of autofs-5.0.7-allow-use-of-hosts-map-in-maps.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index 86e3f48..33a0903 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -1437,7 +1437,8 @@ int parse_mount(struct autofs_point *ap, const char *name,
 			error(LOGOPT_ANY, "options %s", options);
 
 			l = parse_mapent(p, options, &myoptions, &loc, ap->logopt);
-			if (!l) {
+			if ((strstr(myoptions, "fstype=autofs") &&
+			     strstr(myoptions, "hosts")) || !l) {
 				cache_delete_offset_list(mc, name);
 				cache_multi_unlock(me);
 				cache_unlock(mc);
